### PR TITLE
[WFCORE-6972] The standard configuration files used for testsuite/domain should not include community namespaces.

### PR DESCRIPTION
**Pull request by ehsavoie**
The use of namespaces for lower stability levels should be reserved for
stability level specific tests.

Jira: https://issues.redhat.com/browse/WFCORE-6972

### CODE REVIEW

**Comment by ehsavoie** [comment]
Should supersede https://github.com/wildfly/wildfly-core/pull/6153

File: testsuite/domain/src/test/resources/base-server-config.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<server xmlns="urn:jboss:domain:community:20.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:jboss:domain:community:20.0 wildfly-config_community_20_0.xsd">
+<server xmlns="urn:jboss:domain:community:20.0">
```
Comment (yersan) [inline testsuite/domain/src/test/resources/base-server-config.xml:3]: This file is still using "community" name space

Comment (darranl) [reply testsuite/domain/src/test/resources/base-server-config.xml]: This conversation was marked as resolved but I am still also seeing the community namespace here.

Comment (darranl) [reply testsuite/domain/src/test/resources/base-server-config.xml]: Ignore that the GitHub view was showing an outdated diff even though I just opened the PR.

**Review by yersan — APPROVED** [review]

**Comment by yersan** [comment]
Thanks @ehsavoie @darranl

**Pull request merged**